lookout/style/typos/corrector.py

```diff
@@ -120,11 +127,14 @@ def train(self, data: pandas.DataFrame, candidates:  Optional[str] = None,
         :param start_pool_size: Length of data, starting from which multiprocessing is desired.
         :param chunksize: Max size of a chunk for one process during multiprocessing.
         """
+        self._log.info("Training on data with shape %s.", data.shape)
```

Comment:
```suggestion
        self._log.info("Train input: %s of %s", data.shape, data.dtype)
```

Reply:
(it is better to make logs shorter and don't write long NL sentences - they make it longer to understand the logs)

Reply:
dtype is always the same so I'd rather use
```python
self._log.info("Train input shape: %s", data.shape)
```

Reply:
wdym the same? I will call this function on an array of strings and watch it burn in fire.

After all, the purpose of this line is to be a replacement for `repr`, but without the array contents. Let's be consistent.